requirements.txt

```diff
@@ -1,3 +1,4 @@
 networkx>=2.0
 numpy
 pyepics>=3.3.2
+p4p
```

Comment:
Does this belong here so soon? We don't even include caproto in this list yet.

Reply:
Good catch - a mistake on my part. It should be a dev-requirement.